spec/support/helpers/model_spec_helpers.rb

```diff
@@ -0,0 +1,88 @@
+module ModelSpecHelpers
```

Comment:
Can we just make this file about Question stuff? It's already big enough that we shouldn't be adding more helpers from other areas in here.